ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_StateSpecs.rc

```diff
@@ -71,19 +71,19 @@ category: EXPORT
 #........................................................................................
  DUSMASS       | kg m-3     | xy   | N    |                                | Dust Surface Mass Concentration
  DUCMASS       | kg m-2     | xy   | N    |                                | Dust Column Mass Density
- DUEXTTAU      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Extinction AOT
- DUSTEXTTAU    | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Extinction AOT Stratosphere
- DUSCATAU      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Scattering AOT
- DUSTSCATAU    | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Scattering AOT Stratosphere
- DUSMASS25     | kg m-3     | xy   | N    |                                | Dust Surface Mass Concentration - PM 2.5
- DUCMASS25     | kg m-2     | xy   | N    |                                | Dust Column Mass Density - PM 2.5
- DUEXTT25      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Extinction AOT - PM 2.5
- DUSCAT25      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Scattering AOT - PM 2.5
+ DUEXTTAU      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Aerosol Optical Depth
+ DUSTEXTTAU    | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Stratospheric Aerosol Optical Thickness
+ DUSCATAU      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Aerosol Optical Depth Due to Scattering
+ DUSTSCATAU    | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Stratospheric Aerosol Optical Thickness Due to Scattering
+ DUSMASS25     | kg m-3     | xy   | N    |                                | Dust Surface Mass Concentration of Particulate Matter < 2.5 microns (PM 2.5)
+ DUCMASS25     | kg m-2     | xy   | N    |                                | Dust Column Mass Density of Particulate Matter < 2.5 microns (PM 2.5)
+ DUEXTT25      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Aerosol Optical Depth from Particulate Matter < 2.5 microns (PM2.5)
+ DUSCAT25      | 1          | xy   | N    | size(self%wavelengths_vertint) | Dust Aerosol Optical Depth Due to Scattering from Particulate Matter < 2.5 microns (PM2.5)
```

Comment:
perhaps leaving the abbreviation PM or PM2.5 would read better, assuming these terms are common. If we want to be more descriptive, adding 'aerodynamic' or 'geometric' with the diameter would help.